docuploader/__main__.py

```diff
@@ -88,6 +89,23 @@ def upload(
         )
         return sys.exit(1)
 
+    try:
+        if not os.listdir(documentation_path):
+            docuploader.log.error(
+                f"The documentation path given ({documentation_path}) does not contain any documentation files."
+            )
+            return sys.exit(1)
```

Comment:
I don't think you need to `return` when you call `sys.exit`.

Reply:
😓 you're right. Removed all occurrences.